src/VirtoCommerce.Platform.Web/wwwroot/js/app/settings/blades/setting-dictionary.js

```diff
@@ -37,12 +42,9 @@ angular.module('platformWebApp').controller('platformWebApp.settingDictionaryCon
         return false;
     };
 
-    $scope.add = function (form) {
-        if (form.$valid) {
-            currentEntities.push($scope.newValue);
-            resetNewValue();
-            form.$setPristine();
-        }
+    $scope.filteredEntities = function () {
+        loverCasedSearchText = blade.searchText.toLowerCase();
```

Comment:
1. typo in lowerCasedSearchText
2. add var

Reply:
Thanks!